src/org/gk/elv/InstanceZoomablePathwayEditor.java

```diff
@@ -191,7 +188,8 @@ public boolean isUsedAsDrawingTool() {
     private void init() {
         // Link this selection to other displays
         pathwayEditor.getSelectionModel().addGraphEditorActionListener(new GraphEditorActionListener() {
-            public void graphEditorAction(GraphEditorActionEvent e) {
+            @Override
+			public void graphEditorAction(GraphEditorActionEvent e) {
```

Comment:
Hi Liam, Please configure your Eclipse so that your tab is automatically converted into 4 spaces. Otherwise, we see something like this: @Overide and public is not in the same column. By the way, why did you bother to add @Override here? There is no need to spend your time on this.

Reply:
Ah, sorry. In regards to the @Override and removed imports, I think that is my eclipse automatically formatting upon save. I will configure it so those don't occur. And I will change all tabs to 4 spaces!